Update EM graph docs for My Access APIs (#11258)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accesspackageassignmentrequest-post.md

For an administrator to request to remove an assignment, the value of the **requestType** property is `AdminRemove`, and the **accessPackageAssignment** property contains the **id** property identifying the [accessPackageAssignment](../resources/accesspackageassignment.md) being removed.

> For a non-administrator user to request to create an assignment for themselves (either *first assignment* or *renew* assignment to the access package), the value of the **requestType** property is `UserAdd`, and the **accessPackageAssignment** property contains the `targetId` with the ID of the users themselves, the **assignmentPolicyId** property identifying the [accessPackageAssignmentPolicy](../resources/accesspackageassignmentpolicy.md), and the **accessPackageId** property identifying the [accessPackage](../resources/accesspackage.md). The user making the request must already exist in the directory.

Reviewer: ```suggestion
For a non-administrator user to request to create an assignment for themselves (either *first assignment* or *renew* assignment), the value of the *requestType* property is `UserAdd`, and the *accessPackageAssignment* property contains the ID of the users, the *assignmentPolicyId* property identifying the [accessPackageAssignmentPolicy](../resources/accesspackageassignmentpolicy.md), and the *accessPackageId* property identifying the [accessPackage](../resources/accesspackage.md). The user making the request must already exist in the directory.
```

Author: ok

> For a non-administrator user to request to extend an assignment for themselves, the value of the **requestType** property is `UserExtend`, and the **accessPackageAssignment** property contains the `targetId` with the ID of the users themselves, the **assignmentPolicyId** property identifying the [accessPackageAssignmentPolicy](../resources/accesspackageassignmentpolicy.md), and the **accessPackageId** property identifying the [accessPackage](../resources/accesspackage.md). The user making the request must already exist in the directory.

Reviewer: ```suggestion
For a non-administrator user to request to extend their own assignments, the value of the *requestType* property is `UserExtend`. The *accessPackageAssignment* property contains the `targetId` with the `id` of the users. The *assignmentPolicyId* property identifies the [accessPackageAssignmentPolicy](../resources/accesspackageassignmentpolicy.md). The *accessPackageId* property identifies the [accessPackage](../resources/accesspackage.md). The user making the request must already exist in the directory.
```

Author: aligned with this and the previous paragraph

    }]
}
```

> ### Example 3: User requests a package and provide a justification

Reviewer: ```suggestion
### Example 3: Request a package and provide a justification
```

#### Request

> The following is an example of a request where the requestor requested an access package and provided a justification to the approver.

Reviewer: ```suggestion
The following example shows how to request an access package and provide justification to the approver.
```

Author: done
